addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexingContentManagerDecorator.java

```diff
@@ -707,28 +708,27 @@ public Transfer store( final ArtifactStore store, final String path, final Input
                 indexManager.indexTransferIn( transfer, store.getKey() );
             }
 
-            if ( store instanceof Group )
-            {
-                nfc.clearMissing( new ConcreteResource( LocationUtils.toLocation( store ), path ) );
-            }
+            nfc.clearMissing( new ConcreteResource( LocationUtils.toLocation( store ), path ) );
```

Comment:
How much of the NFC and store-affected-by maintenance happening here also happens in the `DefaultContentManager`? If it's only here, that seems dangerous when we disable content indexing...